exporter/signalfxexporter/troubleshooting.md

```diff
@@ -0,0 +1,50 @@
+# Troubleshooting
```

Comment:
As discussed internally, this particular issue should be fixed without any customization required from user. Let's remove this section for now and keep comment fixes to the translation rules only